internal/monitors/filesystems/filesystems.go

```diff
@@ -194,8 +194,17 @@ func (m *Monitor) Configure(conf *Config) error {
 	var ctx context.Context
 	ctx, m.cancel = context.WithCancel(context.Background())
 
-	// save conf to monitor for quick reference
-	m.conf = conf
+	// save shallow copy of conf to monitor for quick reference
+	m.conf = &Config{}
```

Comment:
This line is unnecessary, you are making an empty config instance and then immediately overwriting it on the next line.

Reply:
I'm doing this to avoid dereferencing a nil pointer. Maybe it more readable to do:
confCopy := *conf
m.conf = &confCopy

Reply:
Yes, do that.